### ISSUE TITLE
NVDA doesn't read some checkboxes in miranda's import wizard

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#822]
**Reported by aleksey_s on 2010-08-09 20:36**
str:
1. Go to miranda's menu -> import
2. go through the wizard, select a profile file and choose that you want a custom import
3. tab around the page with checkboxes.
   Current results:
   some of the checkboxes aren't read by NVDA, but they are seen in flat review
   Expected results: NVDA should make use of display information and read those checkboxes.

**Comment by leonardder** [comment]
Reproduced, but closing as won't fix due to 3 reasons:

1. The ticket looks quite abandoned
2. If it wasn't, importing stuff from a profile isn't an every day task, and there is a work around using object navigation, although it requires some counting and going back and forth
3. This should be fixd on Miranda's end, since the way the objects are in the accessibility tree is wrong. Furthermore, it seems they clearly know how to do it right.
